src/overview/onboarding/components/onboarding-step.tsx

```diff
@@ -27,28 +27,42 @@ export default class OnboardingStep extends React.PureComponent<Props> {
     render() {
         return (
             <>
-                <div className={styles.stepContainer}>
-                    <h1 className={this.headerClassName}>
-                        {this.props.titleText}
-                    </h1>
-                    {!this.props.isInitStep && (
-                        <p className={styles.textLarge}>
-                            {this.props.subtitleText}
-                        </p>
-                    )}
-                    {this.props.renderImage()}
-                    <div className={styles.text}>{this.props.children}</div>
-                </div>
-                <div className={styles.navigation}>
-                    {this.props.renderButton()}
-                    {!this.props.isInitStep && (
-                        <ProgressStepContainer
-                            onStepClick={this.props.goToStep}
-                            totalSteps={this.props.totalSteps}
-                            currentStep={this.props.currentStep}
-                        />
-                    )}
-                </div>
+                {this.props.isInitStep ? (
```

Comment:
I feel the readability of this method could be improved a fair bit just by changing this ternary condition operator into a standard `if` statement. It would separate things out a bit, as there are a significant amount of lines for each conditional path. 

eg
```js
render() {
  if (this.props.isInitStep) {
    return (<div className={styles.startingPage}>...)
  }

  return (<div className={styles.whiteBox}>...)
}
```

Reply:
thanks! have fixed